frontend/debug.json

```diff
@@ -0,0 +1,10824 @@
+{
```

Comment:
Do we need this file in the repo?
Should we purge it if not?

Reply:
Was coming from a default environment setting. Removed